src/components/SharePost/SharePost.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Modal from '../Modal/Modal';
+import './SharePost.scss';
+import facebook_logo from '../../assets/images/facebook_logo.png';
+import twitter_logo from '../../assets/images/twitter_logo.png';
+import PropTypes from 'prop-types';
+
+import { FRONTEND_BASE_URL } from '../../constants';
+
+import share from '../../actions/sharePostAction';
+
+import {
+  FacebookShareButton,
+  TwitterShareButton,
+  // LinkedinShareButton,
```

Comment:
@Yoncity, would you also add Linkedin even if it is not yet supported by the backend